### PR TITLE
feat-toast

### PR DESCRIPTION
**Pull request by abhij1607**
image

- feat - toast/snackbar

### CODE REVIEW

File: components/toast.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  text-align: center;
+  border-radius: var(--size-sm);
+  position: fixed;
+  z-index: 1;
+  left: 50%;
+  bottom: var(--size-xl);
+}
+.toast.show {
+  visibility: visible;
+  -webkit-animation: fadein 0.5s, fadeout 0.5s 2.5s;
```
Comment (junaidshaikh-js) [inline components/toast.css]: You don't need to add a browser vendor prefix until you are trying to support a very old browser version

File: docs/toast.html [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+            </li>
+            <li class="nav-list">
+              <a class="link" href="./slider.html">Slider</a>
+            </li>
+            <li class="nav-list">
+              <a class="link link-active" href="./toast.html">Toast</a>
+            </li>
+          </ul>
+        </nav>
+      </aside>
```
Comment (junaidshaikh-js) [inline docs/toast.html:92]: `aside` element can be nested inside the `main`

**Review by junaidshaikh-js — APPROVED** [review]
Great work! 

Good use of semantic elements, and clean code.

File: components/toast.css (hunk 1 above)
Comment (abhij1607) [reply components/toast.css]: ok, thanks for pointing it out

**Review by shivampandey0 — APPROVED** [review]
- Semantic HTML tags used
- Javascript code available to show the demo of components
- CSS variables used

Good to merge

**Comment by abhij1607** [comment]
got 2 good reviews, merging PR

**Pull request merged**